corgidrp/data.py

```diff
@@ -1311,27 +1311,29 @@ class AstrometricCalibration(Image):
     Class for astrometric calibration file. 
     
     Args:
-        data_or_filepath (str or np.array); either the filepath to the FITS file to read in OR the 1D array of calibration measurements
+        data_or_filepath (str or np.array): either the filepath to the FITS file to read in OR an array of calibration measurements (boresight, plate scale, north angle, distortion coeffs)
```

Comment:
can you be a bit more specific on the format of the array if you pass it in? It's ok to use multiple lines to specify this. e.g., which of those values are single floats (platescale, northangle), and which are multiple entries (and what should be the length of them).